server/src/db/seeders/UserSeeder.js

```diff
@@ -0,0 +1,29 @@
+import { User } from "../../models/index.js"
+
+class UserSeeder {
+    static async seed() {
+        const usersData = [
+            {
+                email: "tess.ting@gmail.com",
+                cryptedPassword: "haioejfaheignbkn238"
```

Comment:
setting up the seeder to use `password` means you will be able to log in as these seeded users